### PR TITLE
bug: fix pypi package information

### LINKED ISSUE #937: bug: Wrong pip package information in `envd describe`
## Description

When installing package with `--extra-index-url` such as PyTorch, `--extra-index-url` will also be considered as a separate package

### PR DESCRIPTION
**Pull request by VoVAllen**
Signed-off-by: Jinjing.Zhou <allenzhou@tensorchord.ai>

Fix #937

### CODE REVIEW

**Comment by VoVAllen** [comment]
Open vscode plugsin seems in outage now

File: pkg/types/envd.go [hunk 1]
```diff
@@ -240,12 +241,19 @@ func newDependencyFromLabels(label map[string]string) (*Dependency, error) {
 		}
```
Comment (gaocegege) [inline pkg/types/envd.go:241]: Could we add the unit test for the func?

Comment (VoVAllen) [reply pkg/types/envd.go:250]: added

**Review by gaocegege — APPROVED** [review]
LGTM :+1:
/lgtm

**Pull request merged**